app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/MockNavigationActivity.java

```diff
@@ -125,7 +129,12 @@ protected void onCreate(Bundle savedInstanceState) {
       .navigationNotification(customNotification)
       .build();
 
-    navigation = new MapboxNavigation(this, Mapbox.getAccessToken(), options);
+    navigation = new MapboxNavigation(
+            this,
+            Mapbox.getAccessToken(),
+            options
+    );
+    navigation.setMetricsObserver(this);
```

Comment:
probably not a good idea to pass observer via `MapboxNavigation` since `MetricsReporter` exposes static setter already and the observer can be added by the app via static setter if required